mappings/net/minecraft/block/entity/HangingSignBlockEntity.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_7717 net/minecraft/block/entity/HangingSignBlockEntity
+	FIELD field_40333 HANGING_MAX_TEXT_WIDTH I
+	FIELD field_40334 HANGING_TEXT_LINE_HEIGHT I
```

Comment:
```suggestion
	FIELD field_40333 MAX_TEXT_WIDTH I
	FIELD field_40334 TEXT_LINE_HEIGHT I
```
enigma doesnt like things having the same name as parent classes even though java's fine with it